actions/rules/action_state_to_metastate.go

```diff
@@ -361,9 +361,13 @@ func (act ActionStateToMetaState) onStateChange(newContext change.Detector, cont
 		return nil, nil, err
 	}
 	// next, check which boards are relevant for this WI.
-	groups, err := act.Db.WorkItemTypeGroups().List(act.Ctx, wi.SpaceID)
+	space, err := act.Db.Spaces().Load(act.Ctx, wi.SpaceID)
 	if err != nil {
-		return nil, nil, err
+		return nil, nil, errs.Wrap(err, "error loading space: "+err.Error())
+	}
+	groups, err := act.Db.WorkItemTypeGroups().List(act.Ctx, space.SpaceTemplateID)
+	if err != nil {
+		return nil, nil, errs.Wrap(err, "error loading type groups: "+err.Error())
```

Comment:
Same here, please get rid of `err.Error()`.

Reply:
Fixed. feecf1c6516ea161c05a91182563ba6d822033bd